src/atoms/carousels/carousel.tsx

```diff
@@ -11,6 +11,8 @@ export const Carousel: FC<CarouselProps> = ({ children, onChange, borderRadius }
     onChange={(i) => {
       onChange(i);
     }}
+    preventMovementUntilSwipeScrollTolerance
```

Comment:
on mobile if you try to scroll if you had your finger on carousel it would swap carousel instead of scrolling the page, therefor i added threshold to the carousel to not trigger its swiping unless proper left/right swipe is done